src/class-parsely.php

```diff
@@ -885,9 +885,8 @@ public function construct_parsely_metadata( array $parsely_options, $post ) {
 			$author                   = ( get_query_var( 'author_name' ) ) ? get_user_by( 'slug', get_query_var( 'author_name' ) ) : get_userdata( get_query_var( 'author' ) );
 			$parsely_page['headline'] = $this->get_clean_parsely_page_value( 'Author - ' . $author->data->display_name );
 			$parsely_page['url']      = $current_url;
-		} elseif ( is_category() ) {
-			$category                 = get_the_category();
-			$category                 = $category[0];
+		} elseif ( is_category() || is_archive() ) {
```

Comment:
`is_archive()` is true for many types of archives, like date or tag archives, which are below this conditional, and so requests to a date archive, for instance, will be captured here, instead of below, which will change the behaviour; there may not have been unit tests yet to flag this.

For now, use `is_post_type_archive()` as the most narrowest scope conditional for what you're trying to address here.

Reply:
`is_post_type_archive()` will only be true for the custom post type archive, so what if we did:

`(  is_category() || is_post_type_archive() || is_tax() ) `

Reply:
Yep, that should work 👍🏻